everyvoice/cli.py

```diff
@@ -608,19 +616,19 @@ def demo(
         ["all"],
         "--language",
         "-l",
-        help="Specify languages to be included in the demo. Example: everyvoice demo <path_to_text_to_spec_model> <path_to_spec_to_wav_model> --language eng --language fin",
+        help="Specify languages to be included in the demo. Must be supported by your model. Example: everyvoice demo T2S_MODEL VOCODER --language eng --language fin",
```

Comment:
I don't think I like `T2S_MODEL` since it's too close to TTS and that acronym doesn't appear anywhere else. I'd like to keep `text_to_spec` and `spec_to_wav` since it connects with the naming that we use in our config files and elsewhere in the CLI e.g.:

![image](https://github.com/user-attachments/assets/40515a5f-f756-4fab-93bd-13722c30dfdd)

and `everyvoice train text-to-spec` for example.

You could change to `everyvoice demo [OPTIONS] TEXT_TO_SPEC_MODEL SPEC_TO_WAV_MODEL` if you don't like the angle brackets.

Reply:
Yeah, I hesitate for this one. I didn't like TEXT_TO_SPEC_MODEL  becoming `<path_to_text_to_spec_model>` as it did, but you're right, sticking to the name that occurs makes more sense. I'll fix this.